src/Tenancy/Database/Hooks/DatabaseMutation.php

```diff
@@ -50,6 +50,6 @@ public function fire(): void
 
     public function priority(): int
     {
-        return -100;
+        return 0;
```

Comment:
Can you clarify why -100 didn't work?

Reply:
DatabaseMutation hooks need to be before the MigrationHooks because otherwise, it would result in the Database not existing before the Migration :)
To be really exact, the DatabaseMutation hook should have a priority that always sets it before the Migration Hook. So 0 wouldn't even be correct, but is probably because of the naming (D is before M :)

Reply:
Isn't this rather related to how I sorted the hooks in general. I used sortDesc based on priority. I think there's an issue with that. Let me add a test for this.